packages/web-react/src/components/Matrix/README.md

```diff
@@ -0,0 +1,117 @@
+# Matrix
+
+The Matrix component distributes nested components in equal columns and allows
+their content to be aligned. It's especially useful for multiple stacked layouts
+where content needs to be organized in a uniform grid pattern.
+
+## Basic Usage
+
+```jsx
+<Matrix>{/* Content items */}</Matrix>
+```
+
```

Comment:
Please add the section with compatible components:

https://github.com/lmc-eu/spirit-design-system/tree/main/packages/web/src/scss/components/Matrix#compatible-components